src/dialogs/index.ts

```diff
@@ -0,0 +1,56 @@
+const preload = () => import("./confirmation-dialog");
+
+export interface ConfirmationDialogParams {
+  text?: string;
+  title?: string;
+  confirmText?: string;
+  dismissText?: string;
+  confirm?: () => void;
+  cancel?: () => void;
+  destructive?: boolean;
+}
+
+export const showConfirmationDialog = (
+  dialogParams: ConfirmationDialogParams
```

Comment:
Don't accept a `confirm` and `cancel` here. Only return value should be used.

Reply:
done